modules/utxo-bin/test/parseTransaction.ts

```diff
@@ -5,7 +5,11 @@ import { getFixtureString, getTransactionWithSpendType, formatTreeNoColor } from
 import { ParserNode } from '../src/Parser';
 import { cmdParseTx, getTxParser } from '../src/commands';
 
-utxolib.bitgo.outputScripts.scriptTypes2Of3.forEach((t) => {
+const supportedScriptTypes2Of3 = utxolib.bitgo.outputScripts.scriptTypes2Of3.filter(
+  (scriptType) => scriptType !== 'p2trMusig2'
+);
+
+supportedScriptTypes2Of3.forEach((t) => {
```

Comment:
I'm a bit confused by this - why is p2trMusig2 unsupported?

Reply:
let's add a method `getScriptTypes2of3():` that has a FIXME comment explaining why we skip p2trMusig2

Reply:
Done